registration/forms.py

```diff
@@ -101,7 +101,6 @@ class Meta:
         widgets = {
             'instance_name': TextInput,
             'public_contact_email': EmailInput,
-            'project_description': Textarea,
```

Comment:
Unless it breaks anything, it may be good to keep the widget configuration untouched (`Textarea`) and then just not show the form field.